model/models.go

```diff
@@ -19,6 +19,7 @@ import (
 // FILTERED // the query was filtered by query type
 // NOTFQDN // the query was filtered as it is not fqdn conform
 // SPECIAL // the query was resolved by the special use domain name resolver
+// OTHER // won't be used for EDE option
```

Comment:
Other should be technically be used but we ignore it because of broken clients. So having this for "won't be used" seems like it might make it harder for us to return EDE other in the future (if/when clients are fixed).  
The only spot it's used is in a test. So maybe we change the test to use a value that's not in the enum?  
Like `model.ResponseType(math.MaxInt)`